modules/configuring-layer-three-routed-topology.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: CONCEPT
+[id="configuration-layer-three-routed-topology_{context}"]
+= Configuration for a routed topology
+
+The routed (layer 3) topology networks are a simplified topology for the cluster default network without egress or ingress. In this topology, there is a logical switch per node, each with a different subnet, and a router interconnecting all logical switches.
+
+This configuration can be used for IPv6 and dual-stack deployments.
+
+[NOTE]
+====
+* Layer 3 routed topology networks only allow for the transfer of data packets between pods within a cluster.
+* Creating a secondary network with an IPv6 subnet fails. This is a known limitation and will be fixed a future version of {product-title}.
+====
+
+The following `NetworkAttachmentDefinition` custom resource definition (CRD) YAML describes the fields needed to configure a routed secondary network.
+
+[source,yaml]
+----
+    {
+            "cniVersion": "0.3.1",
+            "name": "ns1-l3-network",
+            "type": "ovn-k8s-cni-overlay",
+            "topology":"layer3",
```

Comment:
For 4.13, the code does not support "topology":"layer3"

Reply:
@weliang1 Hi, 

So this module (modules/configuring-layer-three-routed-topology.adoc) has been hidden from view on docs.openshift.com (indicated by the `//` [here](https://github.com/openshift/openshift-docs/pull/56657/files#diff-b8c3f3e8029c894e473cd22b87c39a4a70b1bf9dbb04e93c46928098bf3f0f16R124)). Users will not see this until I receive approval from @maiqueb to reveal it. 

Here's the preview of content that users will see: https://56657--docspreview.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-additional-networks_configuring-additional-network 


For future reference, should `layer3` be something else? or is it OK as long as users can't see it?

Reply:
it's OK as long as users can't see it